firebase.json

```diff
@@ -1171,7 +1171,9 @@
         "source": "/dart-2.0",
         "destination": "/dart-2",
         "type": 301
-      }
+      },
+      { "source": "/install/**", "destination": "/install", "type": 301 },
```

Comment:
I'm torn about whether to set up a separate redirect for the archive (/install/archive -> /tools/sdk/archive). I suspect that lots of people have links to the archive page, so it seems reasonable to take them to the page they want.

The only reason not to redirect /install/archive to /tools/sdk/archive is if we really DO want to force people to think — to look at the 3 SDK options and then hunt in the Dart SDK page for the archive.

I'm leaning toward redirecting /install/archive to /tools/sdk/archive, to avoid irritating the type of expert user who tends to use the archive.

Reply:
> leaning toward redirecting /install/archive to /tools/sdk/archive

That sounds reasonable.

Reply:
FYI, we'll have both redirects. One for the archive link, and the other to catch the OS specific pages.